### PR TITLE
ci: refactor tpch benchmark pipeline

### PR DESCRIPTION
**Pull request by huangjw806**
I hereby agree to the terms of the [Singularity Data, Inc. Contributor License Agreement](https://gist.github.com/skyzh/0663682a70b0edde7ae991492f2314cb#file-s9y_cla).

## What's changed and what's your intention?

As the title.

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: ci/scripts/benchmark.sh [hunk 1]
```diff
@@ -45,38 +59,63 @@ function cleanup {
...
+DB_PWD=dbpwd
+
 if [[ -z "${RISINGWAVE_IMAGE_TAG+x}" ]]; then
   IMAGE_TAG="latest"
 else
   IMAGE_TAG="${RISINGWAVE_IMAGE_TAG}"
 fi
 
+if [ -z "${BENCH_SKU+x}" ] || [ "${BENCH_SKU}" == "MultiNodeBench" ]; then
+  SKU="multinode"
```
Comment (sumittal) [inline ci/scripts/benchmark.sh:72]: What is the purpose of SKU? can it have only 2 possible values?

**Review by sumittal — COMMENTED** [review]
Added few comments @huangjw806

File: ci/scripts/benchmark.sh (hunk 1 above)
Comment (huangjw806) [reply ci/scripts/benchmark.sh:72]: The SKU is provided by the cloud to create risingwave specifications. Currently, the cloud provides three types of sku: FreeTier, SingleNodeBench, and MultiNodeBench. For bench testing, we only need to create two risingwaves：SingleNodeBench and MultiNodeBench.

File: ci/scripts/benchmark.sh [hunk 2]
```diff
@@ -91,3 +130,6 @@ echo "--- Run Tpch-Bench"
 cd tpch-bench/
 ./scripts/build.sh
 ./scripts/launch_risedev_bench.sh
+
+echo "--- Waiting For Risingwave to Consume Data"
+sleep 300
```
[the remark this answers is not in the document]
Comment (huangjw806) [reply ci/scripts/benchmark.sh:135]: tpch-bench will insert some data into kafka, and sleep 300 can guarantee to consume all the data in kafka.

File: ci/scripts/benchmark.sh [hunk 3]
```diff
@@ -45,38 +59,63 @@ function cleanup {
 
 trap cleanup EXIT
 
+DB_USER=dbuser
+DB_PWD=dbpwd
```
Comment (liurenjie1024) [inline ci/scripts/benchmark.sh:63]: This may leak credentials, use github tokens or sth similar.

Comment (huangjw806) [reply ci/scripts/benchmark.sh:63]: This is just used to run daily,  and the instance will be destroyed when the pipeline ends.

**Review by liurenjie1024 — APPROVED** [review]
LGTM

**Pull request merged**